Fix linking error:

Undefined symbols:
  "_X509_free", referenced from:
      _smtp_open in ssmtp.o
ld: symbol(s) not found


--- configure.in
+++ configure.in
@@ -53,6 +53,6 @@
 if test x$enableval = xyes ; then
 	AC_DEFINE(HAVE_SSL)
-	LIBS="$LIBS -lssl"
+	LIBS="$LIBS -lssl -lcrypto"
 fi
 enableval=""
 
